lib/localize.js

```diff
@@ -129,12 +158,25 @@ export const getLocalizeClass = (superclass = class {}) => class LocalizeClass e
 			const possibleLanguages = this._generatePossibleLanguages(config);
```

Comment:
Change 1: these lines were previously wrapped around an `if (config.importFunc) {` but components that define their own static resources by manually implementing `getLocalizeResources` were then not getting their resources filled in.